manifest-test.yaml

```diff
@@ -0,0 +1,12 @@
+---
+applications:
+- name: funding-service-design-assessment-store-test
+  memory: 64M
+  buildpacks:
```

Comment:
This won't work using the shared workflow as it expects a single `manifest.yml` e.g. https://github.com/communitiesuk/funding-design-service-workflows/blob/main/.github/workflows/deploy.yml#L132  I think we'll need to combine `manifest-dev` and `manifest-test` into a single file with 2 applications for this to work

Reply:
I've merged them into a single file 👾